packages/wrapper/src/vscode/services.ts

```diff
@@ -50,7 +47,20 @@ export const configureServices = async (config: VscodeServicesConfig): Promise<I
             }
         };
     }
-    mergeServices(config.specificServices ?? {}, serviceConfig.userServices);
+    mergeServices(config.specificServices, serviceConfig.userServices);
+
+    // set the log-level via the development settings
+    const devLogLevel = serviceConfig.workspaceConfig.developmentOptions?.logLevel;
+    if (serviceConfig.workspaceConfig.developmentOptions?.logLevel === undefined) {
+
+        // this needs to be done so complicated, because developmentOptions is read-only
+        const devOptions = (serviceConfig.workspaceConfig.developmentOptions as unknown as Record<string, unknown> | undefined) ?? {};
```

Comment:
what about 

```
const devOptions = {
  ...serviceConfig.workspaceConfig.developmentOptions
}
```
?

Reply:
Thank you. It can be written as such now:
```javascript
const devOptions: Record<string, unknown> = {
    ...serviceConfig.workspaceConfig.developmentOptions
};
devOptions.logLevel = config.logLevel;
(serviceConfig.workspaceConfig.developmentOptions as Record<string, unknown>) = Object.assign({}, devOptions);
```

Reply:
Done ⬇️